region/maridia/outer.json

```diff
@@ -3215,9 +3650,32 @@
                       "shinesparkFrames": 65
                     }}
                   ]
+                },
+                {
+                  "name": "Cross Room Jump - Speed Jump",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "maxHeight": 10,
+                      "maxLeftPosition": -38.5,
+                      "minRightPosition": -2
+                    }},
+                    "SpeedBooster",
+                    "canTrickyJump"
```

Comment:
This is a trickyDashJump or uses HiJump

Reply:
Good catch, changed it to canTrickyDashJump now. The HiJump version is already covered by a strat that goes all the way to the top right (3 -> 4).